orchagent/main.cpp

```diff
@@ -149,6 +149,20 @@ int main(int argc, char **argv)
                 exit(EXIT_FAILURE);
             }
             break;
+        case 't':
+        {
+            gMacAgingTimeSet = true;
+            int tmptime = stoi(optarg);
+            if (tmptime < 0)
+            {   
+                macAgingTime = 0;
+                SWSS_LOG_DEBUG("Input macAgingTime < 0, set macAgingTime to 0");
+            }
+            else
+                macAgingTime = tmptime;
+        }
+            break;
```

Comment:
why this break is out of the case's scope?

Reply:
There is no difference for the break to keep inside or outside the brace .
I added the brace only to handle the situation of local variable tmptime.